### ISSUE TITLE
Empty <button> tags

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#126]
The responsive menu has two empty `<button>` tags.

**Comment by imperator99** [comment]
Further investigation shows that both `aria=hidden` and `role=presentation` attributes are added to each of these buttons. This ensures that the empty buttons are hidden to screenreaders and do not affect accessibility. They will, however, continue to fail automated checking. Further development may be required to the external mmenu library (https://github.com/FrDH/jQuery.mmenu/issues/817) to fix if this becomes an ongoing issue.
